quickstart-mongodb-atlas/schemas/mongodb-atlas-cluster.json

```diff
@@ -2,63 +2,125 @@
     "typeName": "MongoDB::Atlas::Cluster",
```

Comment:
Can we update AWS template file examples/cluster/cluster.json

Reply:
fixed